media-source-respec.html

```diff
@@ -2706,6 +2714,367 @@ <h3>Event Summary</h3>
       </section>
     </section>
 
+    <section data-cite="mimesniff" data-dfn-for="ManagedMediaSource">
+      <h2><dfn>ManagedMediaSource</dfn> interface</h2>
+      <p>
+        A {{ManagedMediaSource}} is a {{MediaSource}} that actively manages its
+        memory content. Unlike a {{MediaSource}}, the [=user agent=] MAY evict it
+        from a {{ManagedSourceBuffer}} when under [=ManagedMediaSource/memory
```

Comment:
I think the key part is that the UA may evict memory at any time. E.g., under memory pressure.

Reply:
Are you suggesting a change here?

Reply:
Yes, I think the normative text should effectively say the User Agent may evict memory for any reason, while the non-normative section can give examples like memory pressure, etc.